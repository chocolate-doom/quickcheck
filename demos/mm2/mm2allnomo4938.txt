Doomed Speed Demos Archive
http://doomedsda.us/
-------------------------------------------
Pwad:	MM2.WAD 
Pwad name: Memento Mori 2
Maps:	01-30
Skill:	4
Category:	No Monsters
Exe:      	PRboom+ 2.5.1.4 cl2
	with advanced HUD
-------------------------------------------
Time:     	49:38
-------------------------------------------
Comments:  
I want to kill myself after doing this run. I got lost a lot in episode 1 since I hadn't played it in quite a while, and everything else isn't too bad, but then MAP28 happens. I was going to redo it, but I'm tired of resetting on the MAP09 trick, and unfortunately, my first rusted up attempt was the one that got the trick. I'll redo this again sometime later, most likely, if no one else does.


MAP01 - 0:19 / 0:19  
MAP02 - 0:41 / 1:00  
MAP03 - 1:54 / 2:54 
MAP04 - 1:01 / 3:55  
MAP05 - 1:41 / 5:36  
MAP06 - 3:21 / 8:57
MAP07 - 2:21 / 11:18
MAP08 - 0:52 / 12:10 
MAP09 - 0:21 / 12:31 
MAP10 - 1:09 / 13:40
MAP11 - 1:06 / 14:46  
MAP12 - 0:39 / 15:25  
MAP13 - 1:04 / 16:29  
MAP14 - 1:39 / 18:08  
MAP15 - 1:15 / 19:23  
MAP16 - 0:51 / 20:14
MAP17 - 0:50 / 21:04 
MAP18 - 1:27 / 22:31
MAP19 - 4:04 / 26:35  
MAP20 - 0:47 / 27:22    
MAP21 - 1:26 / 28:48
MAP22 - 3:04 / 31:52  
MAP23 - 0:08 / 32:00
MAP24 - 1:06 / 33:06
MAP25 - 2:00 / 35:06
MAP26 - 2:45 / 37:51
MAP27 - 0:33 / 38:24
MAP28 - 8:27 / 46:51
MAP29 - 1:36 / 48:27
MAP30 - 1:11 / 49:38

-------------------------------------------
Author:    Cyberdemon531
cyberdemon@boun.cr
http://www.twitch.tv/cyberdemon531
https://www.youtube.com/user/Cyberdemon531

