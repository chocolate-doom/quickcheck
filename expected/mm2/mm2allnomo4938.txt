===========================================
MAP01
===========================================

Time: 0:19 (par: 0:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 41 (0%)
	Secrets: 0 / 3 (0%)

===========================================
MAP02
===========================================

Time: 0:41 (par: 1:30)

Player 1 (Green):
	Kills: 0
	Items: 6 / 9 (66%)
	Secrets: 1 / 2 (50%)

===========================================
MAP03
===========================================

Time: 1:54 (par: 2:00)

Player 1 (Green):
	Kills: 0
	Items: 3 / 18 (16%)
	Secrets: 0 / 5 (0%)

===========================================
MAP04
===========================================

Time: 1:01 (par: 2:00)

Player 1 (Green):
	Kills: 0
	Items: 27 / 162 (16%)
	Secrets: 0 / 2 (0%)

===========================================
MAP05
===========================================

Time: 1:41 (par: 1:30)

Player 1 (Green):
	Kills: 0
	Items: 3 / 90 (3%)
	Secrets: 0 / 4 (0%)

===========================================
MAP06
===========================================

Time: 3:21 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 3 / 9 (33%)
	Secrets: 0 / 5 (0%)

===========================================
MAP07
===========================================

Time: 2:21 (par: 2:00)

Player 1 (Green):
	Kills: 0
	Items: 6 / 11 (54%)
	Secrets: 0 / 5 (0%)

===========================================
MAP08
===========================================

Time: 0:52 (par: 2:00)

Player 1 (Green):
	Kills: 0
	Items: 5 / 58 (8%)
	Secrets: 0 / 2 (0%)

===========================================
MAP09
===========================================

Time: 0:21 (par: 4:30)

Player 1 (Green):
	Kills: 0
	Items: 1 / 39 (2%)
	Secrets: 2 / 11 (18%)

===========================================
MAP10
===========================================

Time: 1:09 (par: 1:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 1 (0%)
	Secrets: 0 / 5 (0%)

===========================================
MAP11
===========================================

Time: 1:06 (par: 3:30)

Player 1 (Green):
	Kills: 0
	Items: 5 / 37 (13%)
	Secrets: 0 / 8 (0%)

===========================================
MAP12
===========================================

Time: 0:39 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 9 / 20 (45%)
	Secrets: 0 / 2 (0%)

===========================================
MAP13
===========================================

Time: 1:04 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 4 (0%)
	Secrets: 1 / 5 (20%)

===========================================
MAP14
===========================================

Time: 1:39 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 30 (0%)
	Secrets: 0 / 7 (0%)

===========================================
MAP15
===========================================

Time: 1:15 (par: 3:30)

Player 1 (Green):
	Kills: 0
	Items: 8 / 44 (18%)
	Secrets: 0 / 9 (0%)

===========================================
MAP16
===========================================

Time: 0:51 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 8 / 86 (9%)
	Secrets: 0 / 4 (0%)

===========================================
MAP17
===========================================

Time: 0:50 (par: 7:00)

Player 1 (Green):
	Kills: 0
	Items: 7 / 34 (20%)
	Secrets: 0 / 2 (0%)

===========================================
MAP18
===========================================

Time: 1:27 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 4 (0%)
	Secrets: 0 / 6 (0%)

===========================================
MAP19
===========================================

Time: 4:04 (par: 3:30)

Player 1 (Green):
	Kills: 0
	Items: 10 / 40 (25%)
	Secrets: 0 / 5 (0%)

===========================================
MAP20
===========================================

Time: 0:47 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 18 (0%)
	Secrets: 0 / 4 (0%)

===========================================
MAP21
===========================================

Time: 1:26 (par: 4:00)

Player 1 (Green):
	Kills: 0
	Items: 3 / 16 (18%)
	Secrets: 0 / 6 (0%)

===========================================
MAP22
===========================================

Time: 3:04 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 4 (0%)
	Secrets: 0 / 12 (0%)

===========================================
MAP23
===========================================

Time: 0:08 (par: 3:00)

Player 1 (Green):
	Kills: 0
	Items: 0 / 99 (0%)
	Secrets: 0 / 18 (0%)

===========================================
MAP24
===========================================

Time: 1:06 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 7 / 32 (21%)
	Secrets: 0 / 5 (0%)

===========================================
MAP25
===========================================

Time: 2:00 (par: 2:30)

Player 1 (Green):
	Kills: 0
	Items: 7 / 57 (12%)
	Secrets: 0 / 8 (0%)

===========================================
MAP26
===========================================

Time: 2:45 (par: 5:00)

Player 1 (Green):
	Kills: 0
	Items: 1 / 17 (5%)
	Secrets: 0 / 7 (0%)

===========================================
MAP27
===========================================

Time: 0:33 (par: 5:30)

Player 1 (Green):
	Kills: 0
	Items: 0 / 9 (0%)
	Secrets: 0 / 8 (0%)

===========================================
MAP28
===========================================

Time: 8:27 (par: 7:00)

Player 1 (Green):
	Kills: 0
	Items: 2 / 48 (4%)
	Secrets: 0 / 4 (0%)

===========================================
MAP29
===========================================

Time: 1:36 (par: 5:00)

Player 1 (Green):
	Kills: 0
	Items: 0 / 4 (0%)
	Secrets: 0 / 4 (0%)

===========================================
MAP30
===========================================

Time: 1:11 (par: 3:00)

Player 1 (Green):
	Kills: 0
	Items: 3 / 14 (21%)
	Secrets: 0

